webui/react/src/pages/ExperimentDetails/ExperimentInfoBox.tsx

```diff
@@ -18,9 +21,10 @@ import css from './ExperimentInfoBox.module.scss';
 
 interface Props {
   experiment: ExperimentDetails;
+  fetchExperiment?: () => void;
```

Comment:
**suggestion:** for passing in a callback, how about following the convention of `on[SomeAction]`? Maybe just `onChange`?